package.json

```diff
@@ -25,9 +25,12 @@
     "name": "The AsyncAPI maintainers"
   },
   "scripts": {
-    "start": "npm run build && cross-env NODE_ENV=production node dist/server.js",
+    "start:dev": "npm run build:dev && cross-env NODE_ENV=production node dist/server.js",
```

Comment:
i think we should have dev environment here right 😅?
```suggestion
    "start:dev": "npm run build:dev && cross-env NODE_ENV=development node dist/server.js",
```

Reply:
Thanks!